### PR TITLE
fix(webapp): header cm logo styles

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- fix header cm logo in webapp when the cm user dont have logo and only have the name

image

#Checklist

- [] Check if there is a new color.
- [] Create the color variable in the _colors.scss file.
- [] Follow the color nomenclature.
- [] Create the color class with its variable.
- [] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/templates/_header.scss [hunk 1]
```diff
@@ -50,6 +49,10 @@
         display: block;
         height: 50px;
       }
+
+      span {
+        line-height: 32px;
```
Comment (cbernat) [inline assets/scss/templates/_header.scss:54]: why not 30 here? to make it a round number

Comment (fcoronelmakingsense) [reply assets/scss/templates/_header.scss]: changed

Comment (cbernat) [reply assets/scss/templates/_header.scss]: If 30 is too tight as you've mentioned, leave it that way please.

**Pull request merged**